tools/base64url/base64url.c

```diff
@@ -98,12 +105,12 @@ static unsigned char * get_file_content(const char * file_path, size_t * length)
 }
 
 static unsigned char * get_stdin_content(size_t * length) {
-  int size = 100;
-  unsigned char * out = NULL, buffer[size];
+#define SIZE 100
```

Comment:
Usually I put my `#define` statements on top of the file, for better readability

Reply:
What do you think about the following alternative:
- hard code 100 here,
- and use an `ARRAY_SIZE`-like macro as argument to `read` (or a plain `sizeof`)

Reply:
It's a nice code to look at, but I'd rather not making it more complex than it should be.

I use a simple `#define` statement in those cases and it simplifies the code and easier to debug.

Reply:
Aight. This is the way.